### PR TITLE
fix(patient): re-admission is using existing consultation

### PR DESCRIPTION
**Pull request by cp-Coder**
# Bug Fix

## Proposed Changes

- Added `No consultation filed` for re-admitted patient having no new consultation
- Added consultation button for the above patient types

## Associated Issue

- Fixes https://github.com/coronasafe/care_fe/issues/4125

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug/test a new feature.
- [ ] Update product[documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care)
- [ ] Ensure that UI text is kept in I18n files
- [ ] Prep the screenshot or demo video for the changelog entry, and attach it to issue
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@cp-Coder 
image

the readmitted patient consultation history is not available

File: src/Components/Patient/PatientHome.tsx [hunk 1]
```diff
@@ -332,7 +332,10 @@ export const PatientHome = (props: any) => {
 
   if (isConsultationLoading) {
     consultationList = <CircularProgress size={20} />;
-  } else if (consultationListData.length === 0) {
+  } else if (
+    consultationListData.length === 0 ||
+    (consultationListData.at(0)?.discharge_date && patientData.is_active)
```
Comment (khavinshankar) [inline src/Components/Patient/PatientHome.tsx]: @cp-Coder 
```suggestion
    (consultationListData.at(0)?.discharge_date && patientData.is_active)
```

shouldn't this `.at(0)` be the last consultation??

**Comment by nihal467** [comment]
@cp-Coder what is the status on p1 issues

File: src/Components/Patient/PatientHome.tsx (hunk 1 above)
Comment (aeswibon) [reply src/Components/Patient/PatientHome.tsx]: @khavinshankar, Yes it is the last consultation. If the patient is discharged, the last consultation will have a discharge date.

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**